QUALITY_DECLARATION.md

```diff
@@ -65,7 +65,7 @@ Currently nightly results can be seen here:
 
 ### Feature Documentation [3.i]
 
-`rmw_implementation` does not have any feature documentation and it will be needed to reach higher quality levels.
+`rmw_implementation` features are documented on the repository level [README](README.md).
```

Comment:
```suggestion
`rmw_implementation` features are documented in the repository level [README](README.md).
```

Reply:
Done in https://github.com/ros2/rmw_implementation/pull/103/commits/fb834663b844ac85badeac23b11a4917ace426e3.